ui/main/src/app/modules/feed/components/time-line/init-chart/init-chart.component.ts

```diff
@@ -76,8 +76,16 @@ export class InitChartComponent implements OnInit {
             this.buttonList.push(defaultConfig);
         }
         // Set the zoom activated
-        if (this.buttonList.length > 0) {
-            this.changeGraphConf(this.buttonList[0]);
+        let initialGraphConf = this.buttonList.length > 0 ? this.buttonList[0] : null;
+        
+        const savedDomain = localStorage.getItem('opfab.time-line.domain');
```

Comment:
rename to opfab.timeLine.domain to be consistent with other field (no underscore but uppercase)